keyboards/idobo/config.h

```diff
@@ -20,11 +20,11 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #include "config_common.h"
 
 /* USB Device descriptor parameter */
-#define VENDOR_ID       0xFEED
+#define VENDOR_ID       0x1080 // Idobo - 1d080 - 1080
 #define PRODUCT_ID      0x0000
 #define DEVICE_VER      0x0001
 #define MANUFACTURER    You
```

Comment:
```suggestion
#define MANUFACTURER    Massdrop
```

Apparently this is also known as the "MD75"?
https://drop.com/buy/xd75-aluminum-mechanical-keyboard (from the readme)

Reply:
Massdrop doesn't market this as a Massdrop-produced board. As near as I can tell the actual manufacturer is unknown.

I did also find a purchase listing for it [on AliExpress](https://www.aliexpress.com/i/32952542823.html).

Reply:
This is the https://drop.com/buy/id75-hot-swappable-ortholinear-keyboard-kit
I don't think it's the same as the XD75

Reply:
That's... odd, I could have sworn the link in the readme (and the link I pasted) was this: https://drop.com/buy/75keys-aluminum-mechanical-keyboard-kit

Reply:
> That's... odd, I could have sworn the link in the readme (and the link I pasted) was this: https://drop.com/buy/75keys-aluminum-mechanical-keyboard-kit

Hmm, that's odd indeed. The link you just posted seems to be the same as the XD75.

Yes, I remember seeing a file in the board's folder pointing to XD75 and I've been meaning to send a pull request about it but I forgot.

Reply:
From the comments:
> If you are having issues finding the correct QMK keyboard this is, you can find it as "idobo" in QMK

Reply:
Just fixed the readme

Reply:
In any case, the vendor name should be something other than "You", that's from the templates I think. Maybe "Generic"?

Reply:
Fixed. Thanks for the fast reviews and sry for the multiple commits/not fixing some things in the first place.